core/src/main/scala/com/lightbend/paradox/ErrorContext.scala

```diff
@@ -78,7 +78,13 @@ class ErrorCollector extends ErrorContext {
       .foreach {
         case (page, errors) =>
           // Load contents of the page
-          val lines = scala.io.Source.fromFile(page)("UTF-8").getLines().toList
```

Comment:
Intellij handily picked up the fact that the handle from `source` is never actually closed hence the change here. There are nicer ways of doing this i.e. using Using but this is only available in Scala 2.13